types/fs-extra/index.d.ts

```diff
@@ -263,7 +265,16 @@ export interface CopyOptions {
     overwrite?: boolean;
     preserveTimestamps?: boolean;
     errorOnExist?: boolean;
-    filter?: CopyFilter;
+    filter?: CopyFilterSync | CopyFileterAsync;
+    recursive?: boolean;
+}
+
+export interface CopyOptionsSync {
```

Comment:
You don’t need to re-define the whole options once again. Extend from CopyOptions and just declare the filter here.

Reply:
Yeah, I was debating about that. One could argue that they really aren't an is-a relationship, but I believe the code duplication is less than ideal also. I'll use your suggestion. Thanks for the review!